azkaban-hadoop-security-plugin/build.gradle

```diff
@@ -1,12 +1,20 @@
 apply plugin: 'distribution'
 
+configurations {
+    testCompile.extendsFrom compileOnly
```

Comment:
Modifying the sourceSets is not an issue and is expected. 
I thought extendsFrom would **replace** the existing superConfigs. After some experiments, I realized that it **adds** to the existing configs. 
This change is fine. 
Also since the build only packages the jar and not the run time dependencies, does it need to use compileOnly in the first place? What's your long term plan?